tests/Database/DatabaseEloquentRelationTest.php

```diff
@@ -19,6 +20,18 @@ protected function tearDown(): void
         m::close();
     }
 
+    public function testGetMorphedModelWithEnum()
+    {
+        if (version_compare(PHP_VERSION, '8.1') < 0) {
```

Comment:
Please use annotations for this instead.

Reply:
@driesvints like this?
```
    /**
     * @requires PHP >= 8.1
     */
```

Reply:
Yep

Reply:
done